components/event_dispatcher/introduction.rst

```diff
@@ -212,14 +212,14 @@ instance of ``Symfony\Component\HttpKernel\Event\FilterResponseEvent``::
     and the
     :doc:`DependencyInjection component </components/dependency_injection/introduction>`,
     you can use the
-    :class:`Symfony\\Component\\HttpKernel\\DependencyInjection\\RegisterListenersPass`
-    from the HttpKernel component to tag services as event listeners::
+    :class:`Symfony\\Component\\EventDispatcher\\DependencyInjection\\RegisterListenersPass`
+    from the EventDispatcher component to tag services as event listeners::
```

Comment:
I think we can now remove the "from the EventDispatcher component" part (we are talking about the EventDispatcher component here).